src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -211,6 +211,12 @@
     <DropUnsignedFile Include="$(OutDir)\Microsoft.DebugEngineHost.pdb" />
     <DropUnsignedFile Include="@(RequiredNetFX46FacadeAssemblies)" />
   </ItemGroup>
+  <Target Name="DropMDBFiles" Condition="'$(IsCoreCLR)'=='false'" BeforeTargets="DropFiles">
```

Comment:
You don't actually need a separate target here - that's only necessary if you're checking the IsMonoRuntime variable, which is set by an earlier target.  Otherwise LGTM.